runtests.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+import os
+import subprocess
+import sys
+
+import pytest
+
+STUBS_ROOT = os.path.dirname(os.path.abspath(__file__))
+
+
+def main():
+    subprocess.run(
+        ['pip', 'install', STUBS_ROOT],
```

Comment:
Oh now the more substantial comment: I don't think this works correctly in a virtualenv or conda env. It may install into the base env instead.

Reply:
Can depend on OS and env vars, see e.g. https://stackoverflow.com/questions/8052926/running-subprocess-within-different-virtualenv-with-python

Reply:
Hm ok, switched it to do `[sys.executable, '-m', 'pip', ...]`. It seems like if we were trying to do something more complicated we could still run into problems, but since we're just invoking pip we're ok?

Reply:
That should work I think.